validation/serve.py

```diff
@@ -18,36 +20,43 @@ def get_args():
 app = FastAPI()
 args, _ = get_args()
 
-#################################
-# For testing
-#################################
-# loader = HDF5Loader.HDF5Loader()
-# pcl_data = loader.load_point_cloud_from_h5("bear", "/home/tesha/Documents/Python/404_dream_gaussian/mining/logs/output8")
-# pcl_buffer = loader.pack_point_cloud_to_io_buffer(pcl_data['points'], pcl_data['normals'], pcl_data['features_dc'], pcl_data['features_rest'],
-#                                                   pcl_data['opacities'], pcl_data['scale'], pcl_data['rotation'], pcl_data['sh_degree'])
-# pcl_buffer_encode = base64.b64encode(pcl_buffer.getbuffer())
-#
-# prompt = "an image of the bear"
-# message = TextTo3D(prompt_in=prompt, mesh_out=pcl_buffer_encode)
-
-
-@app.get("/validate")
-# def validate_gs_result(save_images: bool = False):
-def validate_gs_result(message: TextTo3D, save_images: bool = False):
-    print("[INFO] Start validating the input 3D data.")
-    print("[INFO] Input prompt: ", message.prompt_in)
 
+class RequestData(BaseModel):
+    prompt: str
+    data: str
+
+
+class ResponseData(BaseModel):
+    scores: List[float]
```

Comment:
If we give only one 3D object as an input, we should get only score as the output

Reply:
Done